assets/js/phoenix_live_view/live_socket.js

```diff
@@ -771,6 +771,16 @@ export default class LiveSocket {
       if(!phxEvent){ return }
       e.preventDefault()
       e.target.disabled = true
+      if (e.submitter && e.submitter.name) {
+        // it would be cleaner to put e.submitter.name into the args of "push" below, 
+        // without creating a hidden input. That requires changing exec_push (which doesn't
+        // pass args to submitForm), or reusing some parts that it does pass
+        let isubmitter = document.createElement('input')
+        isubmitter.setAttribute('type', 'hidden')
+        isubmitter.setAttribute('name', 'phx_submitted_by')
```

Comment:
What are your thoughts on setting name to `e.submitter.name` and `value` to `e.submitter.value`? That should match the behavior of a "native" form begin submitted, when the button has both name and value properties. c.f. the Actual Behavior section here https://github.com/phoenixframework/phoenix_live_view/issues/184